avocado/core/suite.py

```diff
@@ -277,9 +259,7 @@ def size(self):
     def stats(self):
         """Return a statistics dict with the current tests."""
         runner_name = self.config.get('run.test_runner')
-        if runner_name == 'runner':
-            return self._get_stats_from_runner()
-        elif runner_name == 'nrunner':
+        if runner_name == 'nrunner':
             return self._get_stats_from_nrunner()
```

Comment:
Do we still need `if runner_name == 'nrunner'` here? Also, I think we can rename ` _get_stats_from_nrunner` to just `_get_stats` or just move the stats computing here. What do you think?

Reply:
the if is there because of the same reason it was before. For some reason sometimes this value is not set. I can do a proper refactoring here after we remove the runner, right now I would like to just remove old references.

Reply:
Ok, I am fine with leaving it like it is. can you please create an issue about this problem?